### PR TITLE
Update AuthModal.vue

### PR DESCRIPTION
**Pull request by Semisol**

### CODE REVIEW

File: src/components/AuthModal.vue [hunk 1]
```diff
@@ -34,7 +34,7 @@
                 <ion-spinner v-if="loading" style="height:1.2em;margin-right:5px"></ion-spinner>
                 <ion-label>Sign In</ion-label>
               </ion-button>
-              <span style="font-size:11px;margin-top:10px;">Your data will be securely sent to our servers.</span>
+              <span style="font-size:11px;margin-top:10px;">Your data will only be sent to Scratch and will only be stored on this device. We cannot view your password.</span>
```
Comment (micahlt) [inline src/components/AuthModal.vue:37]: This isn't true though - Scratch will block requests with CORS unless we make them from our servers.  Username and password goes to the Vercel function, then to Scratch.

**Comment by Semisol** [comment]
Well is this not a mobile app? Can't it have a CORS bypass?

**Comment by Semisol** [comment]
Also check modchat rq

File: src/components/AuthModal.vue (hunk 1 above)
Comment (locness3) [reply src/components/AuthModal.vue:37]: And this has already been discussed in #11. @Semisol

**Comment by locness3** [comment]
> Well is this not a mobile app? Can't it have a CORS bypass?

It's a webpage in a webview.

**Comment by Semisol** [comment]
> > Well is this not a mobile app? Can't it have a CORS bypass?
> 
> It's a webpage in a webview.

Well ok then. There **SHOULD** be a way to configure this but if you are going to be lazy then OK. Anyways this needs code changes too but this should be fixed to use the Scratch servers instead of your, **UNTRUSTED** and modifiable to log your password at any time servers.

**Comment by locness3** [comment]
The server is open-source at https://github.com/micahlt/itchy-api. But we should effectively look into bypassing the CORS restrictions; [this article](https://capacitorjs.com/blog/bypassing-cors-with-the-http-plugin) may be a good starting point.

**Comment by Semisol** [comment]
> The server is open-source at [micahlt/itchy-api](https://github.com/micahlt/itchy-api). But we should effectively look into bypassing the CORS restrictions; [this article](https://capacitorjs.com/blog/bypassing-cors-with-the-http-plugin) may be a good starting point.

Backend trust should be eliminated at all costs.  
For real, **open source** is not **trustable**. I hate to say it but so many people think being open source makes it suddenly trustable to give it your password.

**Comment by locness3** [comment]
...but unless Micah's GitHub or Vercel account gets stolen, someone can't modify the running server.

**Comment by Semisol** [comment]
We still cannot know if he does it by himself.

**Comment by Semisol** [comment]
I still would anyways prefer a solution which does not rely on the server.

**Comment by locness3** [comment]
I undertanad and I'll open a new issue for this.

**Pull request closed without merging**